react/utils/constants.ts

```diff
@@ -1 +1,3 @@
 export const IO_DOCUMENTATION = 'vtex.io-documentation@0.x'
+export const CSE_CX = '017314625344810072686:u6y7vlxadyu'
+export const CSE_KEY = 'AIzaSyBgix5AaPDcyu6QbboYdAYjQPsOJtZLOdw'
```

Comment:
Can those keys be public?

Reply:
Yes. We can block the usage by domain name.